tests/aws/services/s3/test_s3.py

```diff
@@ -7603,7 +7603,7 @@ def test_presigned_url_v4_signed_headers_in_qs(
         create_lambda_function(
             func_name=function_name,
             zip_file=testutil.create_zip_file(temp_folder, get_content=True),
-            runtime=Runtime.nodejs16_x,
+            runtime=Runtime.nodejs20_x,
```

Comment:
As mentioned in comments
`we need the AWS SDK v2, and Node 16 still has it by default`
However since AWS has deprecated Node16 we should make this work on Node20